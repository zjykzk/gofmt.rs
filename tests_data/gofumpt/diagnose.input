package p

//gofumpt:diagnose
-- outdated.go --
package p

//gofumpt:diagnose v0.1.0
-- nochange.go --
package p

//gofumpt:diagnosefoobar
