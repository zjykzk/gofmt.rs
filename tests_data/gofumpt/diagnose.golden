package p

//gofumpt:diagnose version: v0.0.0-20220727155840-8dda8068d9f3 (go1.18.29) flags: -lang=go1.16 -modpath=test
-- foo.go.golden-devel --
package p

//gofumpt:diagnose version: (devel) (go1.18.29) flags: -lang=go1.16 -modpath=test
-- foo.go.golden-extra --
package p

//gofumpt:diagnose version: v0.0.0-20220727155840-8dda8068d9f3 (go1.18.29) flags: -lang=go1.16 -modpath=test -extra
-- foo.go.golden-lang --
package p

//gofumpt:diagnose version: v0.0.0-20220727155840-8dda8068d9f3 (go1.18.29) flags: -lang=go1 -modpath=test
-- foo.go.golden-released --
package p

//gofumpt:diagnose version: v0.6.1-0.20240717113859-88a300bbd6dc (go1.18.29) flags: -lang=go1.16 -modpath=test
-- foo.go.golden-external --
package p

//gofumpt:diagnose version: v0.6.1-0.20240717113859-88a300bbd6dc (go1.18.29) flags: -lang=go1.16 -modpath=
